Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -559,6 +559,8 @@ typedef NS_OPTIONS(NSUInteger, WKDisplayCaptureSurfaces) {
 // DERECATED: The setters of the three following function are deprecated, please use overrideLayoutParameters.
 // Define the smallest size a page take with a regular viewport.
 @property (nonatomic, readonly) CGSize _minimumLayoutSizeOverride;
+// Define the smallest size the unobscured area can get for the current view bounds. This value is used to define viewport units.
+@property (nonatomic, readonly) CGSize _minimumUnobscuredSizeOverride;
```

Comment:
This new SPI should also be annotated with `WK_API_AVAILABLE(ios(WK_IOS_TBA))`
```suggestion
@property (nonatomic, readonly) CGSize _minimumUnobscuredSizeOverride WK_API_AVAILABLE(ios(WK_IOS_TBA));
```

Reply:
Fixed in https://github.com/WebKit/WebKit/pull/20271/commits/e626f5a7b66b5e73ff025688869e4eff60b3758a